profiling-ffi/src/exporter.rs

```diff
@@ -169,6 +170,28 @@ unsafe fn into_vec_files<'a>(slice: Slice<'a, File>) -> Vec<exporter::File<'a>>
         .collect()
 }
 
+#[repr(C)]
+pub enum ExporterRequestBuildResult {
```

Comment:
When I did the FFI renames, I chose in some cases not to modify the Rust types, but instead to rename them using cbindgen configuration.

So I think we're missing one of the following:
1. Add `"ExporterRequestBuildResult" = "ddog_prof_Exporter_Request_BuildResult"` to `cbindgen.toml` section `[export.rename]`
2. Directly name this type `Exporter_Request_BuildResult`